deeppavlov/core/commands/train.py

```diff
@@ -40,6 +40,20 @@
 log = get_logger(__name__)
 
 
+def prettify_metrics(metrics, precision=4):
+    """
+    Prettifies the dictionary of metrics
+    """
+    prettified_metrics = OrderedDict()
+    for key, value in metrics:
+        if key.endswith("accuracy"):
+            value = round(100*value, precision)
+        else:
+            value = round(value, precision)
```

Comment:
Rounding is fine, but forcing to multiply accuracy by 100 is questionable. `0.9` and `0.009` are very different.

Reply:
I prefer to see accuracy in percents, it is more readable. How should I fix it.

Reply:
It is not my known accuracy, it is normal accuracy shown in a proper way. I strongly suggest to demonstrate accuracy always in percents.

Reply:
There should be some parameter regulating metrics precision and demonstration, I will think on it.

Reply:
I think it's bad practice to implicitly modify any metrics that ends with 'accuracy'. Multiplication by 100 should be removed.